simapp/app.go

```diff
@@ -443,6 +444,24 @@ func NewSimApp(
 	app.SetBeginBlocker(app.BeginBlocker)
 	app.SetEndBlocker(app.EndBlocker)
 	app.setAnteHandler(encodingConfig.TxConfig, cast.ToStringSlice(appOpts.Get(server.FlagIndexEvents)))
+	// In v0.46, the SDK introduces _postHandlers_. PostHandlers are like
+	// antehandlers, but are run _after_ the `runMsgs` execution. They are also
+	// defined as a chain, and have the same signature as antehandlers.
+	//
+	// In baseapp, postHandlers are run in the same store branch as `runMsgs`,
+	// meaning that both `runMsgs` and `postHandler` state will be committed if
+	// both are successful, and both will be reverted if any of the two fails.
+	//
+	// The SDK exposes a default postHandlers chain, which comprises of only
+	// one decorator: the Transaction Tips decorator. However, some chains do
+	// not need it by default, so the following line is commented out. You can
+	// uncomment it to include the tips decorator, or define your own
+	// postHandler chain. Please note that changing any of the anteHandler or
+	// postHandler chain is likely to be a state-machine breaking change, which
+	// needs a coordinated upgrade.
+	//
+	// Uncomment to enable postHandlers:
+	// app.setPostHandler()
```

Comment:
Any opinions if we should enable Tips by default on simapp (note: all chains that will copy-paste simapp might also enable them)? Is it a use case that's generic enough to be enabled by default on all cosmos chains?

I'd personally say no, and it's more a specific use-case, but would like to hear other opinions.

Reply:
I would go with a yes, when we built craft, one of the discoveries that I think that we had is that an application template with all of the features turned on would be useful. Also, if we have this turned on in simapp, it gets tested

Reply:
But if it’s there and come and to down at least it can be used. Just so that you know, craft began as an implementation of simapp.

basically I’d like to see a Nonsimapp app template in the sdk

I’ve been calling that tiny.

Reply:
For various reasons Gaia is no longer a great template. Unfortunately the same can be said of starport.

Reply:
Like yourself I think it’s not too much of a big deal in either direction if this is present directly in the simulation app or not but I would lean toward including it